hr_timesheet_sheet/models/account_analytic_line.py

```diff
@@ -63,21 +63,28 @@ def _search_sheet(self, operator, value):
 
     @api.multi
     def write(self, values):
-        self._check_state()
+        self._check_state_on_write(values)
         return super(AccountAnalyticLine, self).write(values)
 
     @api.multi
     def unlink(self):
         self._check_state()
         return super(AccountAnalyticLine, self).unlink()
 
+    @api.multi
+    def _check_state_on_write(self, values):
+        """ Hook for extensions """
+        self._check_state()
+
+    @api.multi
     def _check_state(self):
+        if self._context.get('skip_check_state'):
+            return
```

Comment:
why don't you return `True` like before?

Reply:
I removed the return value, since is never used. I think is not needed at all. Anyway I have not any problem to restore it, just let me know.

Reply:
ok, it's fine to leave just as it is. Thanks!